plugins/GSdx/GSOsdManager.h

```diff
@@ -0,0 +1,107 @@
+/*
+ *	Copyright (C) 2014-2016 Gregory hainaut
+ *	Copyright (C) 2016-2016 Jason Brown
+ *
+ *  This Program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2, or (at your option)
+ *  any later version.
+ *
+ *  This Program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with GNU Make; see the file COPYING.  If not, write to
+ *  the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA USA.
+ *  http://www.gnu.org/copyleft/gpl.html
+ *
+ */
+
+#pragma once
+
+#include "stdafx.h"
+#include "GSVector.h"
+#include "GSVertex.h"
+#include "GSTexture.h"
+#include <ft2build.h>
+#include FT_FREETYPE_H
+
+class GSOsdManager {
+	struct glyph_info {
+		int32 ax; // advance.x
+		int32 ay; // advance.y
+
+		uint32 bw; // bitmap.width;
+		uint32 bh; // bitmap.rows;
+
+		int32 bl; // bitmap_left;
+		int32 bt; // bitmap_top;
+
+		float tx; // x offset of glyph
+		float ty; // y offset of glyph
+		float tw; // nomalized glyph width
+	};
+
+	std::map<char32_t, glyph_info> c_info;
```

Comment:
I don't know what "c_" or "k_" mean, maybe a longer name will be nice here. However, for me it would be better to use the "m_" prefix. This way we can easily see the scope of the variable reading the code.